src/components/expanded-state/swap-details/CurrencyTile.js

```diff
@@ -96,7 +95,7 @@ export default function CurrencyTile({
                       </NativePriceText>
                     </Column>
                     <Column alignHorizontal="center" width="content">
-                      <NativePriceText>{symbol}</NativePriceText>
+                      <NativePriceText>{asset?.symbol}</NativePriceText>
```

Comment:
unrelated to the work on this PR, but maybe we want some fallback if for whatever reason symbol is `undefined`?